### ISSUE TITLE
Revise wording when there are no "Detailed Page Stats" available. 

### ISSUE THREAD

**Issue by jamesozzie** [google/site-kit-wp#4097]
## Bug Description

As there are various circumstances whereby a user may encounter no "Detailed Page Stats" for some URLS (see #2862) consider changing the wording of the below:

`It looks like the URL ***** is not part of this site, therefore there is no data available to display`

Examples of where it's possible to encounter this notice when navigating from the Site Kit admin toolbar or when clicking on a URL from the "most popular content" table include the below:
- When using a language plugin (#2862)
- When using AMP or plugins that use query vars (#3080)
- When plugins alter the homepage behaviour (not a single post/page or a category of posts - #3265) 

Any new wording could include something similar to the below:

`It looks like the URL ***** is not part of this site or is not a typical URL based on standard WordPress content types, therefore there is no data available to display. Reach out to the support team or visit our website for support or further information`

## Additional Context

 - PHP Version: 
 - Plugin Version [e.g. 22]

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The "error" message for when Site Kit cannot determine what piece of content the current entity URL in the entity dashboard applies to should be modified as follows: _It looks like the URL xyz is not part of this site or is not based on standard WordPress content types, therefore there is no data available to display. Visit our support forums or website for support or further information_
* The "support forums" bit should link to the [support forums](https://wordpress.org/support/plugin/google-site-kit/), and the "website" bit should link to the [this specific URL on the website](https://sitekit.withgoogle.com/documentation/troubleshooting/dashboard/#url-not-part-of-this-site).

## Implementation Brief

* In `js/components/dashboard-details/DashboardDetailsHeader.js`, Change the text to match the AC with links to both the website and support forums as per AC.
* https://github.com/google/site-kit-wp/blob/develop/assets/js/components/dashboard-details/DashboardDetailsHeader.js#L95-L102

### Test Coverage

* Update any failing tests.

## QA Brief

* Make sure unified dashboard is disabled.
* Vising details page of a non existent URL should show the new copy.
* Make sure that the hyperlink points to the correct links from the AC. 

## Changelog entry

* Revise wording when no entity can be determined based on the given URL in the entity dashboard.

**Comment by felixarntz** [comment]
@bethanylang @jamesozzie If we point users to the website for this, do we want to add a brief documentation article that explains why this could be happening / what the limitations are here?

**Comment by jamesozzie** [comment]
Sounds good to me also. Thanks both.

**Comment by wpdarren** [comment]
### QA Update: ⚠️ 

@felixarntz @bethanylang an observation.  This does pass the AC and QAB, but I noticed comments from both of you regarding creating a guide. At the moment the `website` link points to `https://sitekit.withgoogle.com/`but I feel it would be more user-friendly to point to the URL of the guide that we will create. 

So, my question is do we have a URL for this guide, and if so, does this need to be changed?  Thoughts welcome. 

**Verified:**

- The "error" message for when Site Kit cannot determine what piece of content the current entity URL in the entity dashboard applies is modified as follows: `It looks like the URL xyz is not part of this site or is not based on standard WordPress content types, therefore there is no data available to display. Visit our support forums or website for support or further information`

![image](https://user-images.githubusercontent.com/73545194/142652309-1951e986-905a-45d4-9879-77669f939a4f.png)

- The "support forums" links to the support forums and the "website" links to the website.

**Comment by bethanylang** [comment]
@wpdarren @felixarntz The guide for this is not yet available, so I think we should remove the "website" link and just say "Visit our support forums for assistance." (this needs a period because it's full sentence) and link "support forums" to https://wordpress.org/support/plugin/google-site-kit/. 

Once the guide is ready, I can file a new issue to add the wording and link back. What do you think?

**Comment by felixarntz** [comment]
@bethanylang What's the estimated timeline to get this article ready? This issue is only scheduled for 1.47.0, which is going to be released on December 6. This leaves us with 2 weeks, so wondering if we can get the documentation ready by then, in favor of not removing the link here? If so, we could simply change the URL here once the documentation is there.

**Comment by bethanylang** [comment]
@felixarntz It was scheduled for year-end, but I can put a draft together early next week! The URL will presumably be https://sitekit.withgoogle.com/documentation/google-analytics#url-not-part-of-site.

**Comment by felixarntz** [comment]
@bethanylang Thanks! Why would it be under a `google-analytics` URL though? 🤔

**Comment by bethanylang** [comment]
@felixarntz Ah, I may have misunderstood where this particular message appears – is this Search Console instead of Analytics?

**Comment by felixarntz** [comment]
@bethanylang The message isn't tied to any particular module/service, it shows when Site Kit cannot identify "what" the given URL is - which either means the URL is indeed from another site or some part of the site that's outside of WordPress, or it means the URL is based on some non-WordPress core-supported custom routing logic.

**Comment by bethanylang** [comment]
@felixarntz Got it. In that case I'll need to find a place to put it.. right now, the only non-service section in Troubleshooting is Setup. I guess we could do a Dashboard section?

**Comment by felixarntz** [comment]
@bethanylang That sounds good to me!

**Comment by bethanylang** [comment]
@felixarntz Perfect, will get something ready on Monday.

**Comment by wpdarren** [comment]
@felixarntz I've moved this back to execution since I am assuming we will change the URL?

**Comment by felixarntz** [comment]
@bethanylang Correct - can you provide the URL it will be on here once that's clarified?

**Comment by bethanylang** [comment]
@felixarntz Updated URL will be sitekit.withgoogle.com/documentation/dashboard#url-not-part-of-site. I've drafted the guide and there's an [Asana task for you to review here](https://app.asana.com/0/1200506742078421/1201401763138377).

**Comment by felixarntz** [comment]
@kuasha420 Can you open a quick follow-up PR to replace the website URL here with `https://sitekit.withgoogle.com/documentation/dashboard/#url-not-part-of-site` (see above)? I've updated this in the last bullet point of the ACs as well.

**Comment by kuasha420** [comment]
@felixarntz done for both old and new dashboard. 

cc @eugene-manuilov

**Comment by wpdarren** [comment]
@kuasha420 @eugene-manuilov can I just check that there is a period after  `Visit our support forums or website for support or further information` because there should be one.  I just noticed this when testing #4287

**Comment by kuasha420** [comment]
@wpdarren missed the comment by @bethanylang regarding period. I have added the period to both dashboard in a follow up commit.

**Comment by felixarntz** [comment]
@wpdarren @kuasha420 Nit-picking here, but normally there shouldn't be a period there, the previous approach was correct according to our copy guidelines. I personally understand why you suggested adding a period, but this has come up before and currently our UX guidance is the following:
* Omit the period when it's a single sentence in the UI.
* Omit the period on the second sentence when it's just two sentences.

This falls under the second case - however I wonder whether that guidance is really accurate. We _should_ omit the periods for sure if the second "sentence" is just something like "Learn more" or "Click here to do xyz", but in the case here it's two _actual_ full sentences. Thoughts @marrrmarrr about this?

**Comment by wpdarren** [comment]
Thank you @felixarntz for the explanation.  I am happy to go with the copy guidelines, but since it's a second sentence, I assumed it should have a period.  I know @bethanylang also mentioned this in an earlier comment.

**Comment by wpdarren** [comment]
### **QA Update: ⚠️** 

This ticket is on hold until a decision is made with the period in the second sentence. 

**Verified:**

- The "error" message for when Site Kit cannot determine what piece of content the current entity URL, in the entity dashboard applies is modified as per the AC. 

![image](https://user-images.githubusercontent.com/73545194/143065351-ade55b34-aae1-4589-abbb-381b6882f51c.png)

- The "support forums" links to the forums
- The "website" links to  to `https://sitekit.withgoogle.com/documentation/dashboard/#url-not-part-of-site`

**Note:** the URL above is currently 404 because the page is not live.

**Comment by wpdarren** [comment]
@marrrmarrr hello, please could you add your thoughts to the question left by [Felix here](https://github.com/google/site-kit-wp/issues/4097#issuecomment-976625357) Thanks!

**Comment by marrrmarrr** [comment]
When there are two or more sentences after each other, it's ok to use a full stop. 
For future reference, the complete copywriting guidelines are here: 
https://material.io/design/communication/writing.html#principles 

Quoting the relevant section:

> Periods
> Avoid using periods on solitary sentences within these UI elements:
> - Labels
> - Hover text
> - Bulleted lists
> - Dialog body text

> Use periods on:
> - Multiple sentences
> - Any sentence followed by a link (links themselves should not be full sentences)

**Comment by felixarntz** [comment]
@marrrmarrr Thanks for clarifying!

@wpdarren Okay, based on the above, the period should only be omitted when it's a single sentence or a single sentence followed by a link (e.g. the "Learn more" type of thing). In other words, it's correct to have a period here. I believe other than this everything was QAd without any problems found, so I'm gonna go ahead and move this to Approval.

**Comment by bethanylang** [comment]
@felixarntz and team, sorry to do this to you, but I just got around to putting the documentation together on the staging site and the URL will be slightly different than anticipated. Please use:

/documentation/troubleshooting/dashboard/#not-part-of-this-site

**Comment by felixarntz** [comment]
Sorry @kuasha420, can you please open another quick PR against `main` to update the URL to the above? (I've updated it in the ACs as well.)

**Comment by kuasha420** [comment]
Created PR #4442 to update the URL.

**Comment by wpdarren** [comment]
### QA Update: ✅ 

Verified that the URL is `https://sitekit.withgoogle.com/documentation/troubleshooting/dashboard/#url-not-part-of-this-site` for the website link.  The support link still goes to the support forum
